Source/Blazorise/Utilities/ExpressionConverter.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+
+namespace Blazorise.Utilities
+{
+    /// <summary>
+    /// Helper methods to convert a <see cref="LambdaExpression"/> to valid JavaScript.
+    /// </summary>
+    public static class ExpressionConverter
```

Comment:
I think it's better to have it on the `Blazorise.Charts` project as it is only going to be used there. In case we have a reason we can later move to the core project.

Reply:
If this ExpressionConverter is moved to the `Blazorise.Charts` project, this means that it cannot be used in the `Converters.ToDictionary`. So in that case this converter should stay in the common location. Or do I miss something?

Reply:
You're right, I didn't think it that good 😅

OK it can stay in the core project.